classes/gateways/class.pmprogateway_check.php

```diff
@@ -108,9 +110,18 @@ static function pmpro_payment_option_fields($values, $gateway)
 			</th>
 			<td>
 				<textarea id="instructions" name="instructions" rows="3" cols="50" class="large-text"><?php echo wpautop(  wp_unslash( $values['instructions'] ) ); ?></textarea>
-				<p class="description"><?php esc_html_e('Who to write the check out to. Where to mail it. Shown on checkout, confirmation, and invoice pages.', 'paid-memberships-pro' );?></p>
+				<p class="description"><?php  echo esc_html( sprintf( __( 'Instructions for members to follow to complete their purchase, when paying with %s. Shown on checkout, confirmation, and invoice pages.', 'paid-memberships-pro' ), $check_gateway_label ) );?></p>
 			</td>
-		</tr>	
+		</tr>
+		<tr class="gateway gateway_check" <?php if($gateway != "check") { ?>style="display: none;"<?php } ?>>
+			<th scope="row" valign="top">
+				<label for="check_gateway_label"><?php esc_html_e( 'Gateway Label', 'paid-memberships-pro' );?></label>
+			</th>
+			<td>
+				<input type="text" id="check_gateway_label" name="check_gateway_label" class="regular-text code" value="<?php echo esc_attr( $check_gateway_label ); ?>"/>
+				<p class="description"><?php esc_html_e('Enter a custom payment method that will show on the frontend of your site, please choose a manual payment method like Wire Transfer, Cash or something similar. Defaults to "Pay By Check".', 'paid-memberships-pro' );?></p>
```

Comment:
Do we want the default value to just be "Check" instead of "Pay by Check"? This way, it is consistent to if the site owners change the value to just "Wire Transfer" or "Cash" like this inline documentation recommends.

This would also help clean up text like "Pay with Pay by Check" when using `"Pay with %s"`.